dockerfiles/theia-dev/docker/alpine/install-dependencies.dockerfile

```diff
@@ -16,4 +16,8 @@ RUN apk add --update --no-cache \
     # synchronization tool
     rsync \
     # patch (required in che-theia to apply patches)
-    patch
+    patch \
+    # requirements to run theia with yarn start
+    libsecret \
+    # git tooling for developers
+    tig
```

Comment:
I would not add this library if it's not added in the ubi8 image as well

Reply:
It's not available in the ubi repo's :/
Is someone using the ubi8 image ? does it make sense to maintain it ?

Reply:
@benoitf I have removed tig ... but it is a shame :/